src/Pokedex.jsx

```diff
@@ -71,10 +40,10 @@ const Pokedex = (props) => {
   useEffect(() => {
     axios
       .get(`https://pokeapi.co/api/v2/pokemon?limit=807`)
-      .then(function (response) {
-        const { data } = response;
-        const { results } = data;
+      .then((response) => {
```

Comment:
If we are using ES6+ Javascript, let's use arrow functions always to avoid interchanging the two.